spark-jobs/src/main/scala/za/co/absa/enceladus/spline/persistence/serialization/JSONSerialization.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.spline.persistence.serialization
+
+import org.json4s._
+import org.json4s.jackson.Serialization
+import org.json4s.jackson.Serialization.write
+
+object JSONSerialization {
```

Comment:
Where is this used? `lineage.toJson` in `HadoopFsDataLineageWriter` seems to use the `DataLineage` native method.

Reply:
Great observation, it is actually enough to use the original `za.co.absa.spline.persistence.hdfs.serialization.JSONSerialization` from spline. Fixed.